docs/04-_PalindromDOM.md

```diff
@@ -55,22 +55,30 @@ Property    | Type          | Default    | Description
 ```javascript
 palindrom.method()
 ```
-Attribute   | Type          | Description
+Attribute   | Arguments          | Description
 ---         | ---           | ---
-`unlisten`  | *HTMLElement* | Stop listening to DOM events
-`listen`    | *HTMLElement* | Start listening to DOM events
+`unlisten`  | None | Stop listening to DOM events
+`listen`    | *target: HTMLElement* | Start listening to DOM events
+`async morphUrl`    | *url: String*    | Navigates to a URL by making a request using `async getPatchUsingHTTP` followed by a History API call.
+`async getPatchUsingHTTP`    | *url: String*    | Sends a `PATCH/GET` request to the server demanding a patch that synchronizes server and client sides. It sends a `PATCH` request when there are pending data in the client's queue. And a `GET` request when there is not.
 
 ### Browser history
 
 Palindrom uses the HTML5 history API to update the URL in the browser address bar to reflect the new page. It also listens to a `popstate` event so it could ask the server for new JSON-Patch to morph the page back to previous state. Due to lack of native `pushstate` event you need to either:
- * call `palindrom.changeState(url)` after your `history.pushState(url)`,
- * call `palindrom.morphUrl(url)` - that will call `pushState` and update palindrom's state for you,
- * trigger `palindrom-redirect-pushstate` with `{url: "/new/url"}` on `window` after your `history.pushState(url)`,
+ * call `palindrom.getPatchUsingHTTP(url)` after your `history.pushState(url)`. This method returns a [`Promise<Response>`](https://github.com/axios/axios#response-schema). This methods throws an error if the HTTP request has failed or the `palindrom-before-redirect` was canceled by calling `event.preventDefault()`.
```

Comment:
`This methods throws an error if` -> shouldn't it be "The promise is rejected if..."?

Reply:
Well, the promise is rejected === an error.